src/commands/account/mod.rs

```diff
@@ -33,11 +33,9 @@ pub enum AccountActions {
     ))]
     /// View properties for an account
     ViewAccountSummary(self::view_account_summary::ViewAccountSummary),
-    #[strum_discriminants(strum(
-        message = "import-account          - Import existing account (a.k.a. \"sign in\")"
-    ))]
-    /// Import existing account (a.k.a. "sign in")
-    ImportAccount(self::import_account::Login),
+    #[strum_discriminants(strum(message = "import-account          - Import existing account"))]
+    /// Import existing account
```

Comment:
Let's keep the original message that has "sign in" as "import account" is not immediately obvious to the users.

Reply:
done